k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml

```diff
@@ -4584,8 +4614,8 @@
     "sha256:fe2c106b93db093b30d33d23a6fdb1e163e8e625060fe7aee330baa2b7290b2e": ["v1.20.6"]
 - name: pause
   dmap:
-    "sha256:3d380ca8864549e74af4b29c10f9cb0956236dfb01c40ca076fb6c37253234db": ["3.6"]
     "sha256:1ff6c18fbef2045af6b9c16bf034cc421a29027b800e4f9b68ae9b1cb3e9ae07": ["3.5"]
+    "sha256:3d380ca8864549e74af4b29c10f9cb0956236dfb01c40ca076fb6c37253234db": ["3.6"]
```

Comment:
Just noting that this is a sorting fix, as the digest was added manually in https://github.com/kubernetes/k8s.io/pull/2636 (as opposed to via `cip-mm`).